custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -163,6 +163,19 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
                     projection.scale(scaleCenter.scale)
                         .center(scaleCenter.center)
                         .translate([element.offsetWidth / 2, element.offsetHeight / div]);
+
+                    //setting zoom out limit
+                    //references: https://data-map-d3.readthedocs.io/en/latest/steps/step_06.html#step-06
+                    //            bower_components/angular-datamaps/dist/angular-datamaps.js (line 36)
+                    $(function(){
```

Comment:
Does this work? I would have thought the map was not initialized until after the dom is ready since it's being loaded by angular (often dynamically based on some other user event).

Reply:
This function runs after the entire dom is loaded. Can you mention a case where it loads after user event.( I might have to test this scenario).

Reference: https://stackoverflow.com/a/3880314

Reply:
So for example loading the program summary and then clicking through to the map. Does that also trigger this event?

Reply:
yes. everytime map component loads this event triggers